test/built-ins/Object/hasOwn/hasown_own_getter.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2021 Jamie Kyle.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-object.hasown
+description: Properties - [[HasOwnProperty]] (literal own getter property)
+author: Jamie Kyle
+features: [Object.hasOwn]
+---*/
+
+var o = {
+  get foo() {
+    return 42;
+  }
+};
+
+assert(Object.hasOwn(o, "foo"), 'Object.hasOwn(o, "foo") !== true');
```

Comment:
It might be better to always use `assert.sameValue(Object.hasOwn(o, "foo"), true` in all these tests, to verify that the return value is actually `true` and not just truthy. You are already doing it for `assert.sameValue(Object.hasOwn(o, "foo"), false`.

Reply:
Nevermind, `assert` already does that.

Reply:
oh lol, well I updated it to use `assert.sameValue` with `true`